ccd-definition/AuthorisationCaseEvent/AuthorisationCaseEvent.json

```diff
@@ -560,6 +560,22 @@
       }
     ]
   },
+  {
+    "CaseTypeID": "CIVIL${CCD_DEF_VERSION}",
+    "CaseEventID": "NOTIFY_EVENT",
+    "AccessControl": [
+      {
+        "UserRoles": [
+          "APP-SOL-UNSPEC-PROFILE",
+          "caseworker-civil-admin",
+          "judge-profile",
```

Comment:
Replace "judge-profile", "legal-adviser" roles with  the role  GS_profile so that all the staff and judges will have access to it, remove